hack/tools/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/cespare/prettybench v0.0.0-20150116022406-03b8cfe5406c
 	github.com/client9/misspell v0.3.4
 	github.com/golangci/golangci-lint v1.36.0
-	github.com/google/go-flow-levee v0.1.5-0.20210422150831-99e72693f5c8
+	github.com/google/go-flow-levee v0.1.5
```

Comment:
the pseudo version here was superfluous and we want to cut down on pseudo versions versus tagged releases https://github.com/kubernetes/kubernetes/pull/102697#discussion_r647857512